todoman/cli.py

```diff
@@ -80,6 +80,14 @@ def _validate_start_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_category(ctx, param, val):
+    ctx = ctx.find_object(AppContext)
+    try:
+        return ctx.formatter.parse_category(val)
+    except ValueError as e:
```

Comment:
TBH, I see no point in catching is error. It's just here for posterity's sake _in case_ we make an error in the code elsewhere, but will only catch one error and show another (which won't help either). It's also code that cannot be tested, until we break something else (and even then, we have to manually find that other breakage ourselves, because no tests will).